pyfr/util.py

```diff
@@ -96,6 +96,48 @@ def __exit__(self, *args):
         os.close(self.saved_fds[1])
 
 
+class Interval:
```

Comment:
Can probably drop this class now we have `merge_intervals` and just work with lists of tuples.